schemas/test/en/test_list_collector_repeating_blocks_section_summary.json

```diff
@@ -0,0 +1,361 @@
+                                        {
+                                            "id": "company-or-branch-name",
+                                            "label": "Name of UK company or branch (Mandatory)",
+                                            "mandatory": true,
+                                            "type": "TextField"
+                                        }
+                                    ]
+                                }
+                            },
+                            "remove_block": {
+                                "id": "remove-company",
+                                "type": "ListRemoveQuestion",
+                                "question": {
+                                    "id": "remove-question-companies",
+                                    "type": "General",
+                                    "title": "Are you sure you want to remove this company or UK branch?",
+                                    "answers": [
+                                        {
+                                            "id": "remove-confirmation",
+                                            "mandatory": true,
+                                            "type": "Radio",
+                                            "options": [
+                                                {
+                                                    "label": "Yes",
+                                                    "value": "Yes",
+                                                    "action": {
+                                                        "type": "RemoveListItemAndAnswers"
+                                                    }
+                                                },
+                                                {
+                                                    "label": "No",
+                                                    "value": "No"
+                                                }
+                                            ]
+                                        }
+                                    ]
+                                }
+                            },
+                            "summary": {
+                                "title": "Companies or UK branches",
+                                "item_title": {
+                                    "text": "{company_name}",
+                                    "placeholders": [
+                                        {
+                                            "placeholder": "company_name",
+                                            "value": {
+                                                "source": "answers",
+                                                "identifier": "company-or-branch-name"
+                                            }
+                                        }
+                                    ]
+                                }
```

Comment:
I'd probably sandwich the list collector with a question or two either side, just to check it shows up nicely on the summary page with non list collector related stuff (Spoiler alert, I tried it and it does, and it also works with dynamic answers)

Reply:
I actually had this before and removed because of my manual test fatigue, will add it back in and update any affected tests to expect it.